prime-dotnet-webapi/Services/AccessTermService.cs

```diff
@@ -205,28 +205,38 @@ public async Task<bool> IsCurrentByEnrolleeAsync(Enrollee enrollee)
 
         /// <summary>
         /// Enrollee’s that are:
-        /// Under Review, Locked, Declined -> Current TOA status will read: NA
+        /// Under Review ->  -> Current TOA status: --
+        /// Locked, Declined -> Current TOA status: NA
         /// Required TOA -> Current TOA status: Pending
-        /// Editable -> Current TOA status: Is their signed TOA the most current version and not expired
+        /// Editable (AND before their renewal date) -> Current TOA status: Is their signed TOA the most current version
+        /// Editable (AND on/after their renewal date) -> Current TOA status: --
         /// </summary>
         public async Task<string> GetCurrentTOAStatusAsync(Enrollee enrollee)
         {
             var currentStatus = enrollee.CurrentStatus;
+            var toaStatus = "";
```

Comment:
This code now returns "" instead of "--" when under review

Reply:
Yeah the front end handles it with the default pipe now instead of setting it to "--" on the backend